data/json/professions.json

```diff
@@ -4022,6 +4022,21 @@
       "female": [ "panties" ]
     }
   },
+  {
+    "type": "profession",
+    "id": "breaking_bad_heisenberg",
+    "name": "Chemistry Teacher",
+    "description": "After turning down a top job at a pharma company, you have chosen to use your talents to teach high schoolers.",
+    "points": 0,
+    "skills": [ { "level": 3, "name": "speech" }, { "level": 6, "name": "chemistry" } ],
+    "items": {
```

Comment:
I haven't watched BB, but I'm pretty sure he wasn't nude minus an apron & gasmask 😛

Reply:
It was underwear, gas mask and apron, plus some mittens to handle hot glassware I'm pretty sure?

Reply:
> I haven't watched BB, but I'm pretty sure he wasn't nude minus an apron & gasmask 😛

He was actually, or close to it! It's a big throwback to the series, there are periods where the main character goes to cook meth in the desert and he insists on not wearing his clothes because of how they would smell. So technically his clothes would be in or around the van, but I'm taking some artistic liberty here to make the scenario more interesting